crates/subspace-node/src/chain_spec.rs

```diff
@@ -53,19 +79,59 @@ pub fn testnet_config() -> Result<ChainSpec, String> {
         // ID
         "subspace_test",
         ChainType::Custom("Subspace testnet".to_string()),
-        // TODO: Provide a way for farmer to start with these accounts
         || {
-            testnet_genesis(
+            let sudo_account =
+                AccountId::from_ss58check("5CXTmJEusve5ixyJufqHThmy4qUrrm6FyLCR7QfE4bbyMTNC")
+                    .expect("Wrong root account address");
+
+            // Pre-funded accounts
+            // TODO: Remove these later, this is just for testing
+            let mut balances = vec![
+                (sudo_account.clone(), 1_000 * SSC),
+                (get_account_id_from_seed("Alice"), 1_000 * SSC),
+                (get_account_id_from_seed("Bob"), 1_000 * SSC),
+            ];
+            let vesting_schedules = TOKEN_GRANTS
+                .iter()
+                .flat_map(|&(account_address, amount)| {
+                    let account_id = AccountId::from_ss58check(account_address)
+                        .expect("Wrong vesting account address");
+                    let amount: Balance = amount * SSC;
+
+                    // TODO: Adjust start block to real value before mainnet launch
+                    let start_block = 100_000_000;
+                    let one_month_in_blocks =
+                        u32::try_from(3600 * 24 * 30 * MILLISECS_PER_BLOCK / 1000)
+                            .expect("One month of blocks always fits in u32; qed");
+
+                    // Add balance so it can be locked
+                    balances.push((account_id.clone(), amount));
+
+                    [
+                        // 1/4 of tokens are released after 1 year.
```

Comment:
Unsure if we really want to test the vesting feature on the testnet. If true, we should make the vesting plan faster as we can't wait one year to see it take effect.

Reply:
This is not a test, will explain in DMs.